{
ticks_per_meter: 4900,
encoder_min: -2147483647,
encoder_max: 2147483647,
base_width: 0.5,
transform_tf: false
}
